src/ngMock/angular-mocks.js

```diff
@@ -1788,24 +1788,29 @@ function createHttpBackendMock($rootScope, $timeout, $delegate, $browser) {
    * @ngdoc method
    * @name $httpBackend#flush
    * @description
-   * Flushes all pending requests using the trained responses.
+   * Flushes pending requests using the trained responses in the order they arrived.
+   * If there are no pending requests when the flush method is called
+   * an exception is thrown (as this typically a sign of programming error).
    *
-   * @param {number=} count Number of responses to flush (in the order they arrived). If undefined,
-   *   all pending requests will be flushed. If there are no pending requests when the flush method
-   *   is called an exception is thrown (as this typically a sign of programming error).
+   * @param {number=} count Number of responses to flush. If undefined,
+   *   all pending requests will be flushed beginning at `start`.
+   * @param {number=} [start=0] Sequential request number at which to begin to flush.
```

Comment:
I think `skip` is a better name for this param. It is more intuitive imo.

(Also, have you checked how the resulting docs look like? Can dgeni really understand the `[<name>=<default value>]` syntax?)